ml_perf/reference_implementation.py

```diff
@@ -180,28 +189,31 @@ def validate(state, holdout_glob):
       '--work_dir={}'.format(fsdb.working_dir()))
 
 
-# Evaluate the trained model against the current best model.
-def evaluate(state):
-  eval_model = state.train_model_name
-  best_model = state.best_model_name
+# Evaluate one model against a target.
+def evaluate_model(eval_model, target_model, sgf_dir, seed):
   eval_model_path = os.path.join(fsdb.models_dir(), eval_model)
-  best_model_path = os.path.join(fsdb.models_dir(), best_model)
-  sgf_dir = os.path.join(fsdb.eval_dir(), eval_model)
+  target_model_path = os.path.join(fsdb.models_dir(), target_model)
   result = checked_run('evaluation',
       'bazel-bin/cc/eval',
       '--flagfile={}'.format(os.path.join(FLAGS.flags_dir, 'eval.flags')),
       '--model={}.pb'.format(eval_model_path),
-      '--model_two={}.pb'.format(best_model_path),
+      '--model_two={}.pb'.format(target_model_path),
       '--sgf_dir={}'.format(sgf_dir),
-      '--seed={}'.format(state.seed))
+      '--seed={}'.format(seed))
   result = get_lines(result, make_slice[-7:])
   logging.info(result)
   pattern = '{}\s+\d+\s+(\d+\.\d+)%'.format(eval_model)
   win_rate = float(re.search(pattern, result).group(1)) * 0.01
-  logging.info('Win rate %s vs %s: %.3f', eval_model, best_model, win_rate)
+  logging.info('Win rate %s vs %s: %.3f', eval_model, target_model, win_rate)
```

Comment:
might be a nice place to put a big PASS or FAIL in stdout.

Reply:
Pass or fail depends on the threshold, which isn't know at this point (and is different for gating and the exit criteria)